treex/nn/multi_head_attention copy.py

```diff
@@ -0,0 +1,301 @@
+import typing as tp
```

Comment:
Just a comment, did you mean to add this file?

Reply:
Hey! Nope, was a temp file during a refactor. Thanks for catching that.